protocol/contracts/libraries/Silo/LibSilo.sol

```diff
@@ -316,10 +287,8 @@ library LibSilo {
         }
 
         // emit events.
-        emit LibGerminate.FarmerGerminatingStalkBalanceChanged(
-            account,
-            -int256(stalk)
-        );
+        emit LibGerminate.FarmerGerminatingStalkBalanceChanged(account, -int256(stalk));
+        emit LibGerminate.TotalGerminatingStalkChanged(season, -int256(stalk));
```

Comment:
and here

Reply:
fixed here: https://github.com/BeanstalkFarms/Beanstalk/pull/837/commits/cdc5a0353e9298e5d74888fde498d3a9988114cc#diff-fbd431ce899b0679e919b2bf1fa828e50eeddd381e4efd2012eca396bb7758daR295